.github/workflows/build_tests.yml

```diff
@@ -1,165 +1,110 @@
-        echo "ROCM_PATH=$(hipconfig -R)" >> $GITHUB_ENV
-        echo "gfx90a" | sudo tee --append $(hipconfig -R)/bin/target.lst  # trick ROCm into thinking there's a GPU
-    - name: Echo Environment Variables and Files
-      if: matrix.gpu-api == 'HIP'
-      run: |
-        echo "HIPCONFIG = ${HIPCONFIG}"
-        echo "ROCM_PATH = ${ROCM_PATH}"
-        echo "HIPFFT_PATH = ${HIPFFT_PATH}"
-
-        echo "The contents of $(hipconfig -R)/bin/target.lst are:"
-        sudo cat $(hipconfig -R)/bin/target.lst
-
-    # Perform Build
-    - name: Cholla setup
-      run: |
-        source builds/run_tests.sh
-        setupTests -c gcc
-        echo "CHOLLA_ROOT           = ${CHOLLA_ROOT}"
-        echo "CHOLLA_LAUNCH_COMMAND = ${CHOLLA_LAUNCH_COMMAND}"
-
-        echo "CHOLLA_ROOT=${CHOLLA_ROOT}"                     >> $GITHUB_ENV
-        echo "CHOLLA_LAUNCH_COMMAND=${CHOLLA_LAUNCH_COMMAND}" >> $GITHUB_ENV
-        echo "F_OFFLOAD=${F_OFFLOAD}                          >> $GITHUB_ENV
-        echo "CHOLLA_ENVSET=${CHOLLA_ENVSET}                  >> $GITHUB_ENV
-    - name: Build GoogleTest
-      run: |
-        source builds/run_tests.sh
-        buildGoogleTest
-        echo "GOOGLETEST_ROOT=${GOOGLETEST_ROOT}" >> $GITHUB_ENV
-    - name: Build Cholla
-      run: |
-        source builds/run_tests.sh
-        buildCholla
-    - name: Build Tests
-      run: |
-        source builds/run_tests.sh
-        buildChollaTests
+name: Cholla Compile
+
+on:
+  pull_request:
+  schedule:
+    - cron: "37 07 * * 1"  # run every Monday at 07:37UTC. Crontab computed with crontab.guru
+  workflow_dispatch:
+
+jobs:
+  Build:
+    name: >
+      Build
+      ${{ matrix.container.name }}
+      TYPE=${{ matrix.make-type }}
+
+    # Cuda-toolkit:v${{ matrix.cuda-toolkit-version }}
+    # GCC:v${{ matrix.gcc-version }}
+    # ROCm:v${{ matrix.rocm-version }}
+    # if: ${{ false }}  # If uncommented this line will disable this job
+
+    # Choose OS/Runner
+    runs-on: ubuntu-latest
+    container: 
+      image: ${{matrix.container.link}} 
+    defaults:
+      run:
+        shell: bash
+    # Matrix for different make types
+    strategy:
+      fail-fast: false
+      matrix:
+        make-type: [hydro, gravity, disk, particles, cosmology, mhd]
+        container: [{name: "CUDA", link: "docker://alwinm/cholla:cuda_github"}, {name: "HIP",link: "docker://alwinm/cholla:hip_github"},]
+        # gpu-api: [CUDA]
+        # NOTE: if more than one parameter is in any of these three variables
+        # you need to manually exclude it for the GPU API that doesn't use it.
+        # An example exclude is shown below but commented out. Uncomment and
+        # tweak it to fit your needs
+        # CUDA uses the cuda-toolkit-version and gcc-version
+        # HIP uses the clang-version
+        # cuda-toolkit-version: ['11.2.2']
+        # gcc-version: [9]
+        # rocm-version: ['5.1.0']
+        mpi: ['openmpi'] #Can use mpich and/or openmpi
```

Comment:
I don't think this like is required anymore and the other old matrix stuff should be removed rather than commented out